src/API/sensors/sensor.controller.js

```diff
@@ -0,0 +1,42 @@
+import Sensor from './sensor.model';
+
+function get(req, res, next) {
+    Sensor.get(req.params.id).then((sensor) => {
+        res.json(sensor);
+        return next();
+    }).error((e) => next(e));
+}
+
+function create(req, res, next) {
+    var instance = new Sensor(req.body);
+
+    instance.saveAsync()
+        .then((sensor) => res.json(sensor))
+        .error((e) => next(e));
+}
+
+function update(req, res, next) {
+    Sensor.findByIdAndUpdate(req.params.id, req.body, {
+        new: true
+    })
+        .exec()
+        .then((sensor) => res.json(sensor))
+        .catch((e) => next(e));
```

Comment:
May be simplified to just .catch(next)

Why in previous function you use error instead of catch?